grpc/sa-wrappers.go

```diff
@@ -283,6 +283,26 @@ func (sac StorageAuthorityClientWrapper) PreviousCertificateExists(
 	return exists, err
 }
 
+func (sac StorageAuthorityClientWrapper) AddPrecertificate(
+	ctx context.Context,
+	req *sapb.AddCertificateRequest,
+) (*corepb.Empty, error) {
+	if req == nil {
```

Comment:
I think based on the way the `AddCertificate` wrapper handles a `*sapb.AddCertificateRequest` it also makes sense to be defensive and check the request fields:
https://github.com/letsencrypt/boulder/blob/f02e9da38f0e1e748f678dd3ddfe3bd637ada60b/grpc/sa-wrappers.go#L945-L947

WDYT? I should have been more explicit about that in my prev. review pass, sorry!

Reply:
Actually, this is the client wrapper, so I did the wrong thing here. In the client wrappers we generally verify that the response fields are non-nil, then possibly return `errIncompleteResponse`. Fixing that up.

Relatedly, there's no explicit server-side wrapper, because I embedded the SA implementation on the `StorageAuthorityServerWrapper` so the call goes straight through (in the spirit of reducing dependence on wrappers). However, that means I should double-check the non-nilness in the actual AddPrecertificate / AddSerial functions.

Reply:
Ahhh, ok! I understand. Thanks